Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
Most use cases for either `odie` or `raise` in formulae that I came across were telling the user about conflicting options. Maybe it's worth having a specialized method that is only available to formulae?